framework/src/base/FEProblem.C

```diff
@@ -3582,11 +3573,10 @@ FEProblem::adaptMesh()
   unsigned int cycles_per_step = _adaptivity.getCyclesPerStep();
   for (unsigned int i=0; i < cycles_per_step; ++i)
   {
-    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << "\n";
+    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << std::endl;
```

Comment:
I did this because now meshChanged() can inject Mesh content in this area, so I wanted the buffer to be cleared before dumping more information. Probably not necessary, but it seems like it should be and endl if I am expecting more output.